tools/KoreBuild.Console/Commands/PushToNugetCommand.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.CommandLineUtils;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace KoreBuild.Console.Commands
+{
+    internal class PushToNugetCommand : SubCommandBase
```

Comment:
We probably don't need this command anymore. It's regrettable you've ported this from PS to C# because I was planning to remove it altogether. We don't use it anywhere.